src/app/common/stores/applications/Distribution/Distribution.store.ts

```diff
@@ -153,7 +157,13 @@ class DistributionStore extends BaseStore {
   get userRewardAmount() {
     const walletStore: WalletStore = this.getStore('Wallet');
     // eslint-disable-next-line radix
-    return walletStore.fromWei(parseInt(String(this.rewardAmount)).toString());
+    return Number(walletStore.fromWei(parseInt(String(this.rewardAmount)).toString())) - Number(this.claimedRewards);
+  }
+
+  @computed
+  get userEligibleRewards() {
+    // eslint-disable-next-line radix
+    return Number(this.userRewardAmount) + Number(this.claimedRewards);
```

Comment:
I think the Number here is redundant as this.userRewardAmount already returns a number.

Reply:
done